packages/syft/src/syft/client/client.py

```diff
@@ -121,6 +117,7 @@ def forward_message_to_proxy(
 API_PATH = "/api/v2"
 DEFAULT_PYGRID_PORT = 80
 DEFAULT_PYGRID_ADDRESS = f"http://localhost:{DEFAULT_PYGRID_PORT}"
+INTERNAL_PROXY_TO_RATHOLE = "http://proxy:80/rathole/"
```

Comment:
since proxy is now optional, we'll have to enforce and make sure gateway always enables proxy when tunnel is enabled. Probably might need to revisit this bit in the future.

Reply:
Enforced proxy enable flag: ad31f5bbc924bc3a8b880767cb68dbc2907641a5